configure.in

```diff
@@ -313,6 +315,7 @@ AC_CHECK_LIB(rt, sem_init, , [
 ])
 AC_CHECK_LIB(posix4, sem_wait, , )
 AC_CHECK_LIB(pthread, pthread_create, , )
+AC_CHECK_LIB(flatccrt, flatcc_builder_init, , )
```

Comment:
probably want to enable/disble flatbuffers w/ `--enable-flatbuffers` argument to autoconf?